server/views/applications/pages/health-needs/physical-health.njk

```diff
@@ -2,9 +2,243 @@
 {% set pageName = "physical-health" %}
 
 {% block questions %}
+  {% set phyHealthNeedsFollowUp %}
+  {{
+      formPageTextArea(
+        {
+          fieldName: 'needsDetail',
+          label: {
+            text: page.questions.hasPhyHealthNeeds.needsDetail.question,
+            classes: "govuk-label--s"
+          }
+        },
+        fetchContext()
+      )
+    }}
+  {{
+      formPageRadios(
+        {
+          fieldName: 'canClimbStairs',
+          fieldset: {
+            legend: {
+              text: page.questions.hasPhyHealthNeeds.canClimbStairs.question,
+              classes: "govuk-fieldset__legend--s"
+            }
+          },
+          items: [
+          {
+            value: "yes",
+            text: "Yes"
+          },
+          {
+            value: "no",
+            text: "No"
+          }
+        ]
+        },
+        fetchContext()
+      )
+    }}
+  {% endset %}
 
-  <p class="govuk-body">
-    Physical health page
-  </p>
+  {% set receivingTreatmentFollowUp %}
+  {{
+      formPageInput(
```

Comment:
should this be a `formPageTextArea` rather than an input, looking at the design?

Reply:
same for the other optional inputs below?

Reply:
thank you, yes!